packages/react-card/src/stories/CardScale.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+
+import { Body, Caption } from '@fluentui/react-text';
+
+import { Button } from '@fluentui/react-button';
+import { MoreVerticalRegular } from '@fluentui/react-icons';
+import { Card, CardHeader } from '../index'; // codesandbox-dependency: @fluentui/react-card ^9.0.0-beta
+import { makeStyles, shorthands } from '@griffel/react';
+import { tokens } from '../../../react-theme/src/tokens';
+
+const ASSET_URL = 'https://raw.githubusercontent.com/microsoft/fluentui/master/packages/react-card';
```

Comment:
was wondering what is this for -> turns out we ship this to npm, this should not happen

![image](https://user-images.githubusercontent.com/1223799/155510870-c175d354-49c1-4ead-b0ef-bf046bca3fb7.png)

Reply:
Actually, not only from the stories but also the spec, images describing component behaviors. Should I follow the same thing we did with story assets with a hacky upload to GitHub and use a URL to the asset there?

Reply:
NOTE: this is not the focus of this PR, rather a general alignment that we should discuss with broader team.

Reply:
Created an issue to track this properly https://github.com/microsoft/fluentui/issues/22289